### PR TITLE
Redirect install instructions to standard installfest

### PR DESCRIPTION
**Pull request by CheezItMan**
I've never liked that we had 2 installfests, and will instead have students directed to follow the textbook curriculum installfest.  

I did leave in, for now instructions for them to install chrome and that they'll need to use irb and a text editor over a repl.

### CODE REVIEW

File: preparing-to-code/environment-setup/README.md [hunk 1]
```diff
@@ -52,14 +42,6 @@ To check your version of macOS/OS X, click the Apple icon at the far left edge o
...
 
-#### Xcode Command Line Tools
-
-_Xcode_ is the name of the collection of software tools used to create programs for _macOS_ and _iOS_ (the operating system on Apple's mobile devices like the iPhone and iPad). We'll need some parts of _Xcode_, but not the whole thing. The part we need is called the _Xcode Command Line Tools_. We won't be using them during the Jumpstart curriculum, but it makes sense to install them now. They can take a long time to download and install, so doing it now will save us lots of time in the classroom later.
-
-We'll use the Terminal to install the _Command Line Tools_. Open Terminal and start the installation by typing `$ xcode-select --install`. Follow the instructions and wait for the installation to finish (Protip: have a snack and maybe go for a walk while it's downloading and installing).
-
-After it finishes, verify the installation by typing `xcode-select -p`. It should respond with something very similar to `/Library/Developer/CommandLineTools` (the directory where they were installed).
```
Comment (kaidamasaki) [inline preparing-to-code/environment-setup/README.md]: The installfest refers here for how to install the XCode Command Line Tools.

File: preparing-to-code/environment-setup/README.md [hunk 2]
```diff
@@ -70,52 +52,6 @@ We will use the Chrome web browser by Google in class, because it provides more
...
-* Drag VS Code to your Applications folder and open it up
-* Press `F1` and type `Shell Command: install 'code' command in PATH`.  This will allow you to invoke VS code from the terminal. 
-
-#### Homebrew Package Manager
-
-[Homebrew Package Manager](http://brew.sh) makes it easy to install a variety of software packages, and is used by the following Ruby Version Manager (rvm).  
-
-Installing Homebrew involves copying and pasting `/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
-
-You may need to enter your computer password to complete the installation.
```
Comment (kaidamasaki) [inline preparing-to-code/environment-setup/README.md]: Jumpstart is mentioned for Homebrew as well.

File: preparing-to-code/environment-setup/README.md [hunk 3]
```diff
@@ -70,52 +52,6 @@ We will use the Chrome web browser by Google in class, because it provides more
...
-
-At Ada Developers Academy, we will be using [Visual Studio Code](https://code.visualstudio.com/), a text editor, to write our code.  It is a popular code editor and has a number of features for debugging, and using Git.
-
-##### VS Code installation steps
-
-* Navigate to the [VS Code](https://code.visualstudio.com/) website
-* Download the zip to your Applications folder
-* Unzip the folder, by double-clicking, or right-click _Open_
-* Drag VS Code to your Applications folder and open it up
-* Press `F1` and type `Shell Command: install 'code' command in PATH`.  This will allow you to invoke VS code from the terminal.
```
Comment (kaidamasaki) [inline preparing-to-code/environment-setup/README.md]: VS Code install instructions also point here.

Comment (kaidamasaki) [reply preparing-to-code/environment-setup/README.md]: We should also probably have them use Homebrew to install VS Code.

**Comment by CheezItMan** [comment]
@kaidamasaki Yeah I've updated the textbook installfest to match, and created a PR as well.  It also uses brew to install VS Code.  I'd thought of including instructions to use brew to install Chrome,  but they're more likely to already have it installed.

File: preparing-to-code/environment-setup/README.md [hunk 4]
```diff
@@ -26,21 +26,17 @@ A text editor is a computer program that allows you to create and edit files tha
...
-* [Xcode Command Line Tools](#xcode-command-line-tools)
-* [Google Chrome (web browser)](#google-chrome)
-* [VS Code (text editor)](#vscode)
-* [Homebrew Package Manager](#homebrew-package-manager)
-* [Ruby Version Manager (rvm) (package manager)](#ruby-version-manager)
-* [Ruby (latest stable version of Ruby)](#ruby)
+In the Ada Textbook curriculum we have an [installfest](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/installfest.md) prepared which will walk you through installing all the software required to start at Ada.
 
-#### _macOS High Sierra_, _macOS Sierra_, or _OS X El Capitan_
+#### _macOS Mohave_, _macOS High Sierra_, _macOS Sierra_, or _OS X El Capitan_
```
Comment (kaidamasaki) [inline preparing-to-code/environment-setup/README.md]:
```suggestion
#### _macOS Mojave_, _macOS High Sierra_, _macOS Sierra_, or _OS X El Capitan_
```

**Comment by kaidamasaki** [comment]
@CheezItMan Maybe add a section about Chrome that is just "if you haven't already installed Chrome run...".  (Also, for C13 we want to do Firefox but that will be a bigger change.)

**Comment by CheezItMan** [comment]
> I didn't look at this line by line closely for grammar or clarity, but read through it and it looked good!
> 
> Also, I want to add the comment that: on my new laptop, I installed VS Code with `brew cask install visual-studio-code` and it installed it perfectly into my Applications folder.

I've added sections to install firefox and chrome via homebrew, listing Chrome as optional.  I also added a note on installing Slack via homebrew.

**Pull request merged**